### PR TITLE
add Code of Conduct

### PR DESCRIPTION
**Pull request by sindresorhus**
Adapted from http://contributor-covenant.org

@samccone I added your email as a point of contact since you're the most active maintainer. Let me know if I should change it to someone else.

### CODE REVIEW

**Comment by samccone** [comment]
:+1: 

down to land :airplane:

File: contributing.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Contribute
 
-We're happy to accept contributions in the form of new apps, bug fixes, issues and so on. If you want to help out, add a comment on the issue you want to work on and hack away!
+We're happy to accept contributions in the form of new apps, bug fixes, issues and so on. If you want to help out, add a comment on the issue you want to work on and hack away! Before starting work on an app intended for submission, please open an issue to discuss it with the team. This will allow us to review the framework being used to determine if a spec-compatible app is likely to be accepted.
 
-Note: Before starting work on an app intended for submission, please open an issue to discuss it with the team. This will allow us to review the framework being used to determine if a spec-compatible app is likely to be accepted.
+Please note that this project is released with a [Contributor Code of Conduct](code-of-conduct.md). By participating in this project you agree to abide by its terms.
```
Comment (ericcarraway) [inline contributing.md]: It's largely a matter of taste, but I might consider adding a comma after `project`.  The Grammarly app gives some explanation.
screen shot 2016-01-14 at 6 31 26 pm

Comment (sindresorhus) [inline contributing.md]: @ericcarraway This is just taken from http://contributor-covenant.org/. I'd suggest you open a PR about it there first. https://github.com/CoralineAda/contributor_covenant

Comment (ericcarraway) [inline contributing.md]: :+1:

**Pull request merged**